base/linalg/rowvector.jl

```diff
@@ -103,7 +103,8 @@ convert(::Type{RowVector{T,V}}, rowvec::RowVector) where {T,V<:AbstractVector} =
 """
     transpose(v::AbstractVector)
 
-The transposition operator (`.'`).
+The transposition operator (`.'`). Non-recursive behavior may be obtained in such cases
```

Comment:
Maybe insert `Operates recursively on arrays
 +with vector or matrix element type (e.g., block matrices)` as well here.

Reply:
Note that it *always* acts recursively, independent of element type. We just have `transpose(x::Number) = x`.